samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,100 @@
+.. _ot_ncp_sample:
+
+Thread: NCP sample
+##################
+
+The Thread NCP sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr shell.
+
+Overview
+********
+
+The sample allows to build Thread Border Router target on the MCU. 
+It is important to mention that, according to NCP architecture, MCU part
+needs to cooperate with user higher layer process to altogether establish complete 
+Thread Border Router. Recommended solution is to use docker version of OTBR (OpenThread Border Router)
+and WPAN Network Daemon tool (wpantund), working under Linux operating system, for such purpose 
+(for more information read this: https://openthread.io/guides/border-router).
+
+There is possibility to configure number of OpenThread features, but by default
+application has COMMISSIONER, JOINER, BORDER_ROUTER, BORDER_AGENT and DHCP6_SERVER
+functionalities enabled.
+
+Requirements
+************
+
+* One of the following development kits for testing the network status:
+
+  * |nRF52840DK|
```

Comment:
I think tle look of this section was changed not long ago. Please adjust - have a look at CLI sample

Reply:
Done